components/eam/src/physics/cam/microp_aero.F90

```diff
@@ -496,59 +439,39 @@ subroutine microp_aero_run ( &
 
    itim_old = pbuf_old_tim_idx()
    call pbuf_get_field(pbuf, ast_idx,      ast, start=(/1,1,itim_old/), kount=(/pcols,pver,1/))
-   if(liqcf_fix) then
-      call pbuf_get_field(pbuf, alst_idx,     alst, start=(/1,1,itim_old/), kount=(/pcols,pver,1/))
-      call pbuf_get_field(pbuf, aist_idx,     aist, start=(/1,1,itim_old/), kount=(/pcols,pver,1/))
-   endif
-
-   if(liqcf_fix) then !BSINGH(09/22/2014): It mimics compl treatment
-      liqcldf(:ncol,:pver) = alst(:ncol,:pver) 
-      icecldf(:ncol,:pver) = aist(:ncol,:pver)
-   else
-      liqcldf(:ncol,:pver) = ast(:ncol,:pver)
-      icecldf(:ncol,:pver) = ast(:ncol,:pver)
-   endif
+   call pbuf_get_field(pbuf, alst_idx,     alst, start=(/1,1,itim_old/), kount=(/pcols,pver,1/))
+   call pbuf_get_field(pbuf, aist_idx,     aist, start=(/1,1,itim_old/), kount=(/pcols,pver,1/))
+ 
+   liqcldf(:ncol,:pver) = alst(:ncol,:pver) 
+   icecldf(:ncol,:pver) = aist(:ncol,:pver)
+  
 
    call pbuf_get_field(pbuf, npccn_idx, npccn)
 
-   call pbuf_get_field(pbuf, nacon_idx, nacon)
-   call pbuf_get_field(pbuf, rndst_idx, rndst)
-
-   if (clim_modal_aero) then
 
-      itim_old = pbuf_old_tim_idx()
+   itim_old = pbuf_old_tim_idx()
```

Comment:
This seems repeated....may be we can remove this line.

Reply:
removed